metricbeat/module/redis/redis.go

```diff
@@ -65,8 +65,6 @@ func ParseRedisLine(s string, delimiter string) []string {
 
 // FetchRedisInfo returns a map of requested stats.
 func FetchRedisInfo(stat string, c rd.Conn) (map[string]string, error) {
-	defer c.Close()
```

Comment:
It this deleted on purpose?

Reply:
Yes, I don't think it makes sense to close the connection here.